spec_test.go

```diff
@@ -25,6 +25,9 @@ func TestSpecYamlEncoding(t *testing.T) {
 		IPSANs: []string{
 			"172.31.0.1",
 		},
+		Organizations: []string{
+			"system:masters",
+		},
```

Comment:
@puja108 just to make sure, this is how it works right?

Reply:
Tested for k8s-addons and works. Good for you @xh3b4sd ?